README.md

```diff
@@ -10,3 +10,14 @@ abandoned completely and merged into node_exporter**
 This currently exposes all basic stats (vdev_stats) and most extended stats (vdev_stats_ex) on a
 vdev-level. It doesn't expose per-disk or per-zpool stats, even though these are also available from
 the underlying API.
+
+
+To build with version info:
+
+    docker run -it --rm -v "$PWD":/usr/src/zfs_exporter -w /usr/src/zfs_exporter golang:1.19-bullseye go build -v \
```

Comment:
As much as I personally like building things in containers, I think this should just document the `go build` part of the command line.

Reply:
done